N new.csv

```diff
@@ -0,0 +1,50 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+奶糖,/,noun,1886,,洪士提反譯『萬國藥方』卷一,
+難產,/,noun,1822,LABOUR（Difficult labour，拆副難產；難產. There is a volume on difficult labours, called 達生編 馬禮遜 『英華字典』,
+腦海,/,noun,1913,brains , "A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",
+腦子,/,noun,?,腦子像石頭，總是說不通。,魯迅(1881–1936)『書信集‧致孟十還』, publication date absent in Schmidt
+笑劇,/,noun,?,那些官兒，也一定是笑劇中的腳色。,魯迅(1881–1936)『書信集‧致姚克』, publication date absent in Schmidt
+內部,/,noun,1878,,郭嵩燾『倫敦與巴黎日記』卷廿七,
+內果皮,/,noun,1858,果皮的最後一層,『植物學』,
+內容,/,noun,1839,,林則徐『嚴禁本地人與外人非法往來交易告示』, not clear if the phrase quoted from this source has the same meaning as '事物內部所含的實質或意義。'
+內在,/,noun,1916,immanent,赫美玲 『官話』,
+尿道,/,noun,1869,passage(the passage of the urine),羅存德 『英華字典』,
+凝固,/,noun,1848,TO CONDENSE (to harden), 麥都思 『英華字典』,
+濃度,/,noun,1908,intensity 濃度,顏惠慶 『英華大辭典』,
+暖氣,/,noun,1857,,A. Wylie『六合叢談』, publication date was 1857 but not sure whether it is the publication date of the original source or the translated one
+偶像,/,noun,1857,,A. Wylie『六合叢談』,
+排列,/,noun,1839,,林則徐『英國等躉船鴉片盡數呈繳折』,
+排外主義,/,noun,1908,,顏惠慶 『英華大辭典』,
+牌,/,noun,1822,LICENSE(document giving legal permission),馬禮遜 『英華字典』,
+炮灰,/,noun,1934,殺人者在毁壞世界，救人者在修補它，而炮灰 資格的諸公，卻總在恭維殺人者。,魯迅『且介亭雜文‧拿破侖與隋那』,
+朋友,/,noun,1837,,K. F. A. Gützlaff『東西洋考每月統記傳』,
+批判,/,verb,1927,criticize,豐子愷 『豐子愷文集』,I hold the question whether the quotation is reliable here
```

Comment:
```suggestion
批判,/,verb,1927,criticize,豐子愷『豐子愷文集』,I hold the question whether the quotation is reliable here
```
Overall, the entries are much more readable now. But there’s these small things to fix as well as missing punctuation marks in a couple of places.